### ISSUE TITLE
Question on C1 part value

### ISSUE THREAD

**Issue by dburr** [SukkoPera/OpenC64Saver#6]
I am unable to find 0.2uf capacitors, however I do have 0.22uf ones. Can I substitute these instead, without it adversely affecting the performance?

**Comment by SukkoPera** [comment]
Yes, 220nF is actually the correct common commercial value, there's no ponit in chasing down an impossible-to-find 200nF cap.

**Comment by SukkoPera** [comment]
Well, reopening so that one day in the next 50 years I remember to fix this.
